lib/code_corps/github/api/eager_api.ex

```diff
@@ -0,0 +1,83 @@
+defmodule CodeCorps.GitHub.EagerAPI do
+  @moduledoc """
+  Eager loads a resource from the GitHub API by fetching all of its pages in
+  parallel.
+  """
+
+  def get_all(url, headers, options) do
+    HTTPoison.start
+    {:ok, response} = HTTPoison.get(url, headers, options)
```

Comment:
This should likely be modified to call into a more central place so we can mock the `%HTTPoison.Response{}` (headers included). Otherwise it will be hard to test pagination.